File: code/java/src/main/java/iterator/Post.java
package iterator;

class Post {
    private String title;
    private String author;
    private String summary;

    public Post(String title, String author, String summary) {
        this.title = title;
        this.author = author;
        this.summary = summary;
    }

    // Getters for title, author, and summary
    public String getTitle() {
        return title;
    }

    public String getAuthor() {
        return author;
    }

    public String getSummary() {
        return summary;
    }
}

File: code/python/iterator/Post.py
class Post:
    def __init__(self, title, author, summary):
        self.title = title
        self.author = author
        self.summary = summary

    def get_title(self):
        return self.title

    def get_author(self):
        return self.author

    def get_summary(self):
        return self.summary
